client/packages/coldchain/src/Equipment/DetailView/StatusForm.tsx

```diff
@@ -4,19 +4,41 @@ import {
   BasicTextInput,
   InputWithLabelRow,
   Typography,
+  Upload,
 } from '@common/components';
 import { LocaleKey, useTranslation } from '@common/intl';
 import {
   AssetLogStatusInput,
   Box,
   InsertAssetLogInput,
   StatusType,
+  styled,
   useDebounceCallback,
+  useIsScreen,
 } from '@openmsupply-client/common';
-import { AssetLogPanel } from '../Components';
+import { FileList } from '../Components';
 import { parseLogStatus } from '../utils';
 import { useAssetData } from '@openmsupply-client/system';
 import { useIsGapsStoreOnly } from '@openmsupply-client/common';
+import { TakePhotoButton } from './TakePhotoButton';
+import { Capacitor } from '@capacitor/core';
+
+const StyledContainer = styled(Box)(({ theme }) => ({
+  borderColor: theme.palette.divider,
+  flexDirection: 'column',
+  display: 'flex',
+  alignItems: 'center',
+  height: '100%',
+  width: '100%',
+}));
+
```

Comment:
This is styling previously in AssetLogPanel which we no longer need (as we no longer have tabs in the modal)